dev-libs/pcl/pcl-1.12.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+DESCRIPTION="Portable Coroutine Library"
+HOMEPAGE="http://xmailserver.org/libpcl.html"
+SRC_URI="http://xmailserver.org/pcl-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~x86 ~amd64 ~arm"
```

Comment:
I've installed the pcl library on my Funtoo amd64 and Funtoo Raspberry Pi 2 and run the examples, I think it is ok to add them to the KEYWORDS